week-4-practise/level-2/exercise-1.js

```diff
@@ -11,13 +11,13 @@ We made a start on this for you here:
 var bottle = {
   volume: 0,
   fill: function() {
-    // calling this function should make you bottles volume = 100;
+    return (this.volume = this.volume + 100); // calling this function should make you bottles volume = 100;

```

Comment:
You don't actually need to return anything here. More importantly though, this function will overfill the bottle - filling it to the top should leave it at 100, like 100%. If it's half full this takes it to 150...